processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -121,8 +121,8 @@
 
   // TDigest sketch aggregators
   public static final byte TDIGEST_BUILD_SKETCH_CACHE_TYPE_ID = 0x38;
-  public static final byte TDIGEST_MERGE_SKETCH_CACHE_TYPE_ID = 0x39;
   public static final byte TDIGEST_SKETCH_TO_QUANTILES_CACHE_TYPE_ID = 0x40;
```

Comment:
Since SKETCH_TO_QUANTILES and SKETCH_TO_QUANTILE are postaggs, their IDs should go in `PostAggregatorIds` instead